src/Fmt_ast.ml

```diff
@@ -2383,7 +2414,7 @@ and fmt_class_expr c ?eol ?(box = true) ({ast= exp} as xexp) =
            $ fmt_class_type c (sub_cty ~ctx t) ))
       $ fmt_atrs
   | Pcl_extension ext -> fmt_extension c ctx "%" ext $ fmt_atrs
-  | Pcl_open (flag, lid, cl) ->
+  | Pcl_open ({popen_override= flag; popen_expr= lid; _}, cl) ->
```

Comment:
handle `popen_loc` and `popen_attributes` ..

Reply:
Same as in class types